packages/sdk/src/blockFinder.ts

```diff
@@ -0,0 +1,103 @@
+import assert from "assert";
```

Comment:
If you're going to copy this, why not have financial-templates-lib pull it from here? Or have the users of that method just pull it from here? Would prefer not to have multiple copies of the same code if possible.

Reply:
ok so youa re saying update financial templates to pull from sdk?  i can do that

Reply:
i updated ftl to import blockfinder from sdk